### PR TITLE
Accept path to model or a TF IOHandler

### PR DESCRIPTION
**Pull request by mlaco**
Groundwork that will allow NSFWJS to ship with a built-in model by using @mlaco's TF model loader.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -35,25 +35,36 @@ export async function load(base = BASE_PATH, options = { size: IMAGE_SIZE }) {
...
+    modelPathBaseOrIOHandler: string | IOHandler,
+    options: nsfwjsOptions
+  ) {
     this.options = options
-    this.path = `${base}model.json`
     this.normalizationOffset = tf.scalar(255)
+
+    if (typeof modelPathBaseOrIOHandler === 'string') {
+      this.pathOrIOHandler = `${modelPathBaseOrIOHandler}model.json`
+    } else {
```
Comment (GantMan) [inline src/index.ts]: this line is a bit confusing.  Can you delete it?

Comment (mlaco) [reply src/index.ts]: Nope, this was supposed to set `pathOrIOHandler` in the case of `IOHandler`

Comment (GantMan) [reply src/index.ts]: lol, I was tired too, bc I didn't notice it wasn't set, hehehe.

**Comment by GantMan** [comment]
giphy

**Pull request merged**